public/assets/css/styles.scss

```diff
@@ -439,4 +439,45 @@ svg {
       stroke-dashoffset: 0;
     }
   }
+}
+
+/* Custom CSS */
+.offcanvas {
```

Comment:
Let's use styled-components to define the component styles instead of adding them to styles.scss
This is an old file and we need to migrate old styles step by step

Reply:
done